db_upgrade_scripts/mosip_audit/upgrade.properties

```diff
@@ -7,6 +8,5 @@ SYS_ADMIN_USER=
 SYS_ADMIN_PWD=
 DEFAULT_DB_NAME=postgres
 DBUSER_PWD=
-CURRENT_VERSION=
-TARGET_VERSION=
-ACTION=
\ No newline at end of file
+CURRENT_VERSION=1.1.5.5
+UPGRADE_VERSION=1.2.0.1-B1
```

Comment:
this is a common file, should we prefix the versions ?

Reply:
removed